src/server/raft_client.rs

```diff
@@ -285,7 +285,18 @@ impl Buffer for BatchMessageBuffer {
 
     #[inline]
     fn flush(&mut self, sender: &mut ClientCStreamSender<BatchRaftMessage>) -> grpcio::Result<()> {
-        let batch = mem::take(&mut self.batch);
+        let mut batch = mem::take(&mut self.batch);
```

Comment:
Consider changing `self.batch` to `Vec<(Instant, RaftMessage)>`, so we won't need to add a new field to the RaftMessage protobuf.
To accomplish this, you might want to modify `Queue<RaftMessage>` to `Queue<(Instant, RaftMessage)>`.

Reply:
@overvenus Thanks for your advice! I have finished it. PTAL again, thx~